harness/tests/experiment/pytorch/test_pytorch_trial.py

```diff
@@ -583,6 +610,42 @@ def make_workloads() -> workload.Stream:
         )
         controller.run()
 
+    # At the time of writing this test, supports_mixed_precision and
+    # supports_averaging_training_metrics are class methods, but we instantiate a
+    # PyTorchTrialController here in case that ever changes. We expect that a controller
+    # instantiated in the following way should always provide these supports.
+    def test_supports(self) -> None:
```

Comment:
I don't think we call `supports_mixed_precision` anywhere anymore.  We used to, to ensure somebody didn't configure mixed precision for a trial that didn't support it.

But now we are moving to Trainer APIs, and these sorts of things will get set in a parameter of the framework-specific Trainer API, and if somebody passes `mixed_precision=True` to a non-mixed-precision-capable trainer api, they'll get an error about illegal parameter names.

So I wouldn't add this test.  I would delete `supports_mixed_precision` and I would even consider deleting `supports_averaging_training_metrics` (though probably I'd hold off).

But ultimately, getters that return booleans don't need tests.  There are bigger fish to fry.

Reply:
@drh-determined-ai, you still haven't responded to this.

Adding tests to test tech debt is basically just adding tech debt, and I really think you should not add this test to the source code.

Reply:
Sorry. I lost track of this thread somehow. I have removed the test.